client_src/js/lib/modules/printerSettings.js

```diff
@@ -1120,29 +1125,29 @@ class PrinterSettings {
     pageElements.mainPage.socket.className = `btn btn-${currentPrinter.webSocketState.colour} mb-1 btn-block`;
 
     pageElements.connectPage.apiCheck.innerHTML = `<i class="fas fa-link"></i> <b>API Check</b><br><b>Last Checked: </b>${Calc.dateClean(
-      currentPrinter.systemChecks.scanning.api.date
```

Comment:
I am going to remove the systemChecks alltogether. The generation of caches is not uniformly coupled to a printer (JobCache being an exception) and they are not "cleaning" anymore. There is no failure anymore, nor do I try-catch it as the TaskManager does this automatically.

The only success or failure is whether to have a websocket connection and this is already visualized with the 3 states: Host API, WebSocket and Printer Connection. Whether to have responses for Files, Profile, Filament (if applicable) and Settings is of course still important. Those problems should not be hidden in a printer settings panel, but in a general OctoFarm notification/event system which can report problems processing data from OctoPrint.

To conclude, the only reason systemChecks exist was because to copy data into a different format. Remove the different format altogether and there's no need for a system check or cleaning.